buildspec.yml

```diff
@@ -43,11 +43,11 @@ phases:
       - echo "//npm.pkg.github.com/:_authToken=${GITHUB_ACCESS_TOKEN}" > ~/.npmrc
       - echo Setting environment variables
       - cd .aws
-      - npm install
       - tfenv install
+      - npm ci
       - npm run build
       # synthesize the js into terraform json with the proper node environment
-      - 'if [ "$GIT_BRANCH" == "$DEV_BRANCH" ]; then NODE_ENV=development npm run synth; else npm run synth; fi'
```

Comment:
shit, thanks for catching this, i missed a few across repos. damn dash vs bash vs ash inconsistencies.